packages/desktop-client/src/components/PrivacyFilter.tsx

```diff
@@ -63,46 +61,70 @@ export function PrivacyFilter({
         typeof value === 'boolean' ? value : value(),
       ));
 
-  const blurAmount = blurIntensity != null ? `${blurIntensity}px` : '3px';
-
   return !activate ? (
     <>{Children.toArray(children)}</>
   ) : (
-    <BlurredOverlay blurIntensity={blurAmount} {...props}>
-      {children}
-    </BlurredOverlay>
+    <PrivacyOverlay {...props}>{children}</PrivacyOverlay>
   );
 }
 
-function BlurredOverlay({ blurIntensity, children, ...props }) {
-  const [hovered, setHovered] = useState(false);
-  const onHover = useCallback(() => setHovered(true), [setHovered]);
-  const onHoverEnd = useCallback(() => setHovered(false), [setHovered]);
-
-  const blurStyle = {
-    ...(!hovered && {
-      filter: `blur(${blurIntensity})`,
-      WebkitFilter: `blur(${blurIntensity})`,
-      // To fix blur performance issue in Safari.
-      // https://graffino.com/til/CjT2jrcLHP-how-to-fix-filter-blur-performance-issue-in-safari
-      transform: `translate3d(0, 0, 0)`,
-    }),
-  };
-
+function PrivacyOverlay({ children, ...props }) {
   const { style, ...restProps } = props;
 
   return (
     <View
-      style={{
-        display: style?.display ? style.display : 'inline-flex',
-        ...blurStyle,
-        ...style,
-      }}
-      onPointerEnter={onHover}
-      onPointerLeave={onHoverEnd}
+      className={`${css(
+        [
+          {
+            display: 'inline-flex',
+            flexGrow: 1,
+            position: 'relative',
+            ' > div:first-child': {
+              opacity: 0,
+            },
+            ' > div:nth-child(2)': {
+              display: 'flex',
+            },
+            '&:hover': {
+              ' > div:first-child': {
+                opacity: 1,
+              },
+              ' > div:nth-child(2)': {
+                display: 'none',
+              },
+            },
+          },
+        ],
+        style,
+      )}`}
       {...restProps}
     >
-      {children}
+      <div
+        className={`${css([
+          {
+            display: 'flex',
+            flexGrow: 1,
+          },
+        ])}`}
+      >
+        {children}
+      </div>
+
+      <div
+        aria-hidden="true"
+        className={`${css({
+          flexDirection: 'column',
+          fontFamily: 'Redacted Script',
+          height: '100%',
+          inset: 0,
+          justifyContent: 'center',
+          pointerEvents: 'none',
+          position: 'absolute',
+          width: '100%',
+        })}`}
+      >
+        {children}
+      </div>
```

Comment:
Guessing coderabbit missed that the font is a package dependency. IMO we must assume it will be available and will load, as we do for the other dependencies.

Maintainers?